doc/whats_new.rst

```diff
@@ -330,6 +330,11 @@ Enhancements
      (`#7248 <https://github.com/scikit-learn/scikit-learn/pull/7248>_`)
      By `Andreas Müller`_.
 
+   - Support sparse contingency matrices in cluster evaluation
+     (:mod:`metrics.cluster.supervised`) and use these by default.
```

Comment:
"uses these by default" is not entirely true, right? Only in the internals, but no return value changed.